### PR TITLE
acceptance test for incorrect login credentials

### PR DESCRIPTION
**Pull request by HariBhandari07**
## Description

Add acceptance tests for incorrect login credentials
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
CI
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Trashbin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/6155/

<p>














</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Files</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/6155/

<p>


















</p></details>

File: tests/acceptance/stepDefinitions/loginContext.js [hunk 1]
```diff
@@ -10,12 +10,18 @@ Given(/^the user has browsed to the login page$/,
...
+Given('the user has clicked the authenticate button', () => client.page.loginPage().authenticate())
+
 When('the user clicks the authenticate button', () => client.page.loginPage().authenticate())
 
 When('the user logs in with username {string} and password {string} using the webUI',
   (username, password) => client.page.ownCloudLoginPage().login(username, password)
 )
 
+When('the user tries to log in with username {string} and password {string} using the webUI',
+  (username, password) => client.page.ownCloudLoginPage().login(username, password)
```
Comment (dpakach) [inline tests/acceptance/stepDefinitions/loginContext.js]: As it seems `ownCloudLoginPage().login()`  just clicks the login button after filling in the form, Shouldn't we also wait for some ajax calls or something. This way when we later check that the user is redirected to login page then we can verify that the user has been redirected there after login request was actually sent.

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Trashbin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/6206/

<p>





























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Files</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/6206/

<p>










</p></details>

**Pull request merged**